Release v10.0.0-beta.17 (#259)
mapbox/mapbox-maps-ios: CHANGELOG.md

### Bug fixes 🐞

> * Fixes an issue that could cause issues with annotations including causing them to not be selectable ([#246](https://github.com/mapbox/mapbox-maps-ios/pull/246))

Reviewer: Maybe
```suggestion
* Fixes an issue that could prevent annotations from being selectable. ([#246](https://github.com/mapbox/mapbox-maps-ios/pull/246))
```

* Fixes an issue where some JSON layers are not be decoded correctly ([#248](https://github.com/mapbox/mapbox-maps-ios/pull/248))

> * Fixes an issue where the location puck was not animating ([#256](https://github.com/mapbox/mapbox-maps-ios/pull/256))

Reviewer: Add periods to the end of changelog entries for consistency.
